src/components/item/header/Actions.tsx

```diff
@@ -77,34 +79,47 @@ const Actions = ({ item }: Props): JSX.Element | null => {
         <MoreVert />
       </IconButton>
       <Menu id={internalId} anchorEl={anchorEl} open={open} onClose={closeMenu}>
-        <CreateShortcutButton
-          key="shortcut"
-          onClick={() => {
-            openCreateShortcutModal();
-            closeMenu();
-          }}
-        />
-        <BookmarkButton
-          size="medium"
-          key="bookmark"
-          type={ActionButton.MENU_ITEM}
-          item={item}
-        />
-        {canWrite && (
+        {member.type === AccountType.Individual ? (
           <>
-            <Divider />
-            <HideButton key="hide" type={ActionButton.MENU_ITEM} item={item} />
-            <PinButton key="pin" type={ActionButton.MENU_ITEM} item={item} />
-            {item.type !== ItemType.FOLDER && (
-              <CollapseButton
-                key="collapse"
-                type={ActionButton.MENU_ITEM}
-                item={item}
-              />
+            <CreateShortcutButton
+              key="shortcut"
+              onClick={() => {
+                openCreateShortcutModal();
+                closeMenu();
+              }}
+            />
+            <BookmarkButton
+              size="medium"
+              key="bookmark"
+              type={ActionButton.MENU_ITEM}
+              item={item}
+              className={ITEM_MENU_BOOKMARK_BUTTON_CLASS}
```

Comment:
Selection using className ? Can we use an id instead ? I think we should move away from classes for selectors.

Reply:
We sometimes use classnames because `id` doesn't work. Here I think it's because the graasp-ui component does not allow me to set an id, but allows a class names.
Maybe we should resolve this issue alttogether from ui.